src/eth/TransactionManager.js

```diff
@@ -172,10 +172,16 @@ class Tracker {
     this._transactions[key].push(tx);
 
     for (let event of this.constructor.states) {
-      tx.on(event, () => this._listeners[key][event].forEach(cb => cb(tx)));
+      tx.on(event, () =>
+        this._listeners[key][event].forEach(
+          cb => (tx.error ? cb(tx, tx.error.message) : cb(tx))
```

Comment:
If there is an error property on the transaction, send the error message back with the callback function.